### PR TITLE
Add projector.update action. Add tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/system/action/projector/test_update.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        self.set_models({"projector/111": {"name": "name_srtgb123"}})
+        response = self.request(
+            "projector.update",
+            {
+                "id": 111,
+                "name": "name_Xcdfgee",
+                "width": 100,
+                "aspect_ratio_numerator": 3,
+                "aspect_ratio_denominator": 4,
+                "color": "ffffff",
```
Comment (jsangmeister) [inline tests/system/action/projector/test_update.py:15]: All colors have to be prefixed with a `#`. Please also add tests that check that non-compliant color strings are not accepted (the implementation for that is in #482).

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/projector/test_update.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/projector/test_update.py]: Add the prefix. If #482  is merged, I can add a non-correct color string.

File: openslides_backend/action/actions/projector/update.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "color",
+            "background_color",
+            "header_background_color",
+            "header_font_color",
+            "header_h1_color",
+            "chyron_background_color",
+            "chyron_font_color",
+            "show_header_footer",
+            "show_title",
+            "show_logo",
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/update.py:29]: `show_clock` is missing

Comment (FinnStutzenstein) [reply openslides_backend/action/actions/projector/update.py:29]: also in the tests

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/projector/update.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/update.py:29]: Add `show_clock` and update test.

**Comment by reiterl** [comment]
Add a test for wrong color. Rebased to have the color fields.

**Review by FinnStutzenstein — APPROVED** [review]

**Review by jsangmeister — APPROVED** [review]

**Comment by FinnStutzenstein** [comment]
@reiterl Please resolve conflicts

**Pull request merged**